saleor/plugins/webhook/utils.py

```diff
@@ -104,3 +128,25 @@ def parse_payment_action_response(
             "transaction_already_processed", False
         ),
     )
+
+
+def parse_list_shipping_methods_response(
```

Comment:
This method is EasyPost specific case. I think that you return here all details which you received  from EP API. This is not something which we can easily extend in the future. The received response should be direclty mapped to the structure which this method returns. Currently ExternalShippingMethod contains below fields. And these fields should we look when we parse an response.  
We should not do any customization (like we do currently for name or ID, - it is fine that we add `app` to the beginning, but we shouldn't relay on custom fields). 

```
@dataclass
class ExternalShippingMethod:
    """Dataclass for storing information about a shipping method."""
    id: str
    name: str
    price: Money
```
See `saleor.plugins.webhook.utils.parse_payment_action_response` as an example of mapping. We map all fields from the response 1:1 to our dataclass. 
In your case EP should return a response aligned to Saleor's generic response structure, not an EP response which Saleor will map to own structure. In that case, we can't easily integrate any different type of Shipping provider because as a response we expect that we will get EP response.